site/www/index.xslt

```diff
@@ -170,7 +172,7 @@ show(p)
                 function reveal(e) {
                     cachedCode.style.display = "block";
                     cachedImg.style.display = "none";
-                    cachedText.textContent = "hide code";
+                    cachedText.textContent = "show plot";
```

Comment:
I can't figure out why this change was made or where it's used.

Reply:
![Screen Shot 2020-08-18 at 5 00 36 PM](https://user-images.githubusercontent.com/13773586/90565136-64b10480-e174-11ea-9436-fb828587959b.png)
It's the text under the image of the code / plot.

Reply:
Maybe I'm not looking in the right place. Can you send me the exact link to this? I don't see it on the homepage or on the input unification page clicking on Learn More.